content/docs/1.13/apis.md

```diff
@@ -0,0 +1,81 @@
+
+For legacy reasons, the Agent also accepts spans in Zipkin format, however, only very old versions of Jaeger clients can send data in that format and it is officially deprecated.
+
+### Protobuf via gRPC (stable)
+
+In a typical Jaeger deployment, Agents receive spans from Clients and forward them to Collectors. Since Jaeger version 1.11 the official and recommended protocol between Agents and Collectors is gRPC with Protobuf. The Protobuf IDL [collector.proto][collector.proto] is currently located in the main Jaeger repository, under [model/proto/api_v2][collector.proto]. In the future it will be moved to [jaeger-idl][jaeger-idl] repository ([jaeger-idl/issues/1234](TODO-link)).
+
+### Thrift over HTTP (stable)
+
+In some cases it is not feasible to deploy Jaeger Agent next to the application, for example, when the application code is running as AWS Lambda function. In these scenarios the Jaeger Clients can be configured to submit spans directly to the Collectors over HTTP/HTTPS.
+
+The same [jaeger.thrift][jaeger.thrift] payload can be submitted in HTTP POST request to `/api/traces` endpoint, e.g. https://jaeger-collector:14268/api/traces. The `Batch` struct needs to be encoded using Thrift's `binary` encoding, and the HTTP request should specify the content type header:
+
+```
+Content-Type: application/vnd.apache.thrift.binary
+```
+
+### JSON over HTTP (n/a)
+
+There is no official Jaeger JSON format that can be accepted by the collector. In the future the Protobuf-generated JSON may be supported.
+
+### Thrift via TChannel (deprecated)
+
+Agent and Collector can communicate using TChannel protocol. This protocol is generally not supported by  the routing infrastructure and has been deprecated. It will be eventually removed from Jaeger.
+
+### Zipkin Formats (stable)
+
+Jaeger Collector can also accept spans in several Zipkin data format, namely JSON v1/v2 and Thrift. The Collector needs to be configured to enable Zipkin HTTP server, e.g. on port 9411 used by Zipkin collectors. The server enables two endpoints that expect POST requests:
+
+* `/api/v1/spans` for submitting spans in Zipkin JSON v1 or Zipkin Thrift format.
+* `/api/v2/spans` for submitting spans in Zipkin JSON v2 or format.
+
+## Trace retrieval APIs
+
+Traves saved in the storage can be retrieved by calling Jaeger Query Service.
+
+### gRPC/Protobuf (stable)
+
+The recommended way for programmatically retrieving traces is via gRPC endpoint defined in [query.proto][query.proto] IDL file (located in the main Jaeger repository, similar to [collector.proto][collector.proto]).
+
+### HTTP JSON (restricted)
+
+Jaeger UI communicates with Jaeger Query Service via JSON API. For example, a trace can be retrieved via GET request to https://jaeger-query:16686/api/traces/{trace-id-hex-string}. This JSON API is intentionally undocumented and subject to change (see [jaeger/issues/1234](TODO-link)).
+
+## Clients configuration
```

Comment:
If the structure is changed to be based on components, then this could be moved to the Agent section. Should also have a status - guess it is stable?

Reply:
I  wasn't sure how to classify this one. Technically, it's stable & backwards compatible, but it's also kind-of "not documented"